includes/Checker/Abstract_Check_Runner.php

```diff
@@ -74,6 +74,14 @@ abstract class Abstract_Check_Runner implements Check_Runner {
 	 */
 	protected $runtime_environment;
 
+	/**
+	 * Whether to include experimental checks.
+	 *
+	 * @since n.e.x.t
+	 * @var bool
+	 */
+	protected $include_experimental = null;
```

Comment:
Nit-pick, but the other similar class properties aren't initialized like this, so we can remove it from here too. Uninitialized class properties are `null` anyway.
```suggestion
	protected $include_experimental;
```

Reply:
Thanks @felixarntz I've updated this